src/main/Core/ExtensionManager/ExtensionManagerModule.ts

```diff
@@ -73,6 +74,10 @@ export class ExtensionManagerModule {
             },
         );
 
+        ipcMain.on("getEnvironmentVariable", (event, { environmentVariable }) => {
+            event.returnValue = environmentVariableProvider.get(environmentVariable);
+        });
```

Comment:
Why did you place this in the `ExtensionManagerModule`? This has nothing to do with extensions. I suggest moving it to the appropriate module.

Reply:
Oops, forgot to move it after testing. Will fix rn